### PR TITLE
Streams

### PR DESCRIPTION
**Pull request by joeflack4**
Updates
```
    Streams
    - Finished work to allow for more streams to be queried. Initialized by adding: 'implementers', 'snomed'.
```

### CODE REVIEW

**Comment by rohaher** [comment]
I’ll take a look

**Comment by joeflack4** [comment]
@rohaher My hours on this project are going to be greatly reduced :(, so I'm trying to wrap up some work w/ this PR.

**Comment by rohaher** [comment]
Ah okay :( let me know if u want me to pick up where u leave off

**Comment by joeflack4** [comment]
@rohaher Thanks, I'm gonna try to finish this today, but it'll be by end of tomorrow the latest.

Yeah, I'll basically just be playing the role of helping you from now on. Probably some of the project scope can be reduced. We'll touch base w/ Davera later in the week ~

File: fhir_zulip_nlp/fhir_zulip_nlp.py [hunk 1]
```diff
@@ -48,9 +48,10 @@
 # TODO: change CSV names to clarify that `zulip_raw_results.csv` is only for queried / matched messages
 RAW_RESULTS_MATCHES_FILENAME = 'zulip_raw_results.csv'
 RAW_RESULTS_ALL_FILENAME = 'zulip_raw_results_all.csv'
+STREAM_NAME_FIELD = 'display_recipient'  # strange, but this is the 'stream name' field from Zulip API
 CONFIG = {
     'zuliprc_path': os.path.join(ENV_DIR, '.zuliprc'),  # rc = "runtime config"
-    'streams': ['terminology'],
+    'streams': ['terminology', 'implementers', 'snomed'],
```
Comment (joeflack4) [inline fhir_zulip_nlp/fhir_zulip_nlp.py:54]: @rohaher Here's where the different streams are declared.

Ideally I'd add this as an option in the google sheet. Maybe one day.

**Review by joeflack4 — COMMENTED** [review]
Most of this was just adding an extra for loop in some of these functions to iterate over streams.

**Pull request merged**